tests/integration_tests/framework/utils.py

```diff
@@ -107,16 +107,20 @@ def create_rest_client(host, **kwargs):
         cert=cert_path)
 
 
-def create_pika_connection(host):
+def create_pika_connection(container_id):
+    host = docker.get_manager_ip(container_id)
+    ca_data = docker.read_file(
+        container_id,
+        '/etc/cloudify/ssl/cloudify_internal_ca_cert.pem')
+
     credentials = pika.credentials.PlainCredentials(
         username='cloudify',
         password='c10udify')
+    ssl_ctx = ssl.create_default_context(cadata=ca_data)
     return pika.BlockingConnection(
         pika.ConnectionParameters(host=host,
                                   port=5671,
-                                  ssl_options=pika.SSLOptions(
-                                      ssl.create_default_context(
-                                          cafile=get_broker_ssl_cert_path())),
```

Comment:
we of course can't use that function, because that path only exists inside the container, and we're calling it from outside